CMakeLists.txt

```diff
@@ -167,12 +167,10 @@ else()
   target_link_libraries(rmm PUBLIC CUDA::cudart)
 endif(CUDA_STATIC_RUNTIME)
 
-target_link_libraries(rmm PUBLIC CUDA::cuda_driver)
-
```

Comment:
This is good -- I had added this to keep previous behavior for the time being, but I was already thinking that it wasn't actually being used.